docs/application-development/features.md

```diff
@@ -3082,6 +3082,196 @@ const handle = await client.start(scheduledWorkflow, {
 </TabItem>
 </Tabs>
 
+## Side Effects
+
+A Side Effect is a method of execution to produce nondeterministic code, such as generating a UUID or a random number.
```

Comment:
Not a method of execution. Doesn't produce code.

Side Effects are a way to execute non-deterministic code and don't compromise the determinism of the workflow by storing the non-deterministic result into workflow history and getting the same exact value during Replay.